src/plugins/ui-selector/public/js/ui-selector.js

```diff
@@ -0,0 +1,57 @@
+(function() {
```

Comment:
The UI Selector needs a commend or description to clarify it is for selecting which UI theme should be applied to the browser.  Its intent was confusing at first glance.